Test.java

```diff
@@ -0,0 +1,164 @@
+
+public class Test {
+
+    static Team galatasaray = new Team(1, "galatasaray", "GS");
+    static Team fenerbahce = new Team(2, "fenerbahce", "FB");
+    static Team besiktas = new Team(3, "besiktas", "BJ");
+    static Team sivasspor = new Team(4, "sivasspor", "SS");
+    static Team trabzonspor = new Team(5, "trabzonspor", "TS");
+    static Team göztepe = new Team(6, "göztepe", "GT");
+    static Team antalyaspor = new Team(7, "antalyaspor", "AS");
+    static Team gaziantepspor = new Team(8, "gaziantepspor", "GA");
+    static Team genclerbirligi = new Team(9, "genclerbirligi", "GB");
+    static Team bursaspor = new Team(10, "bursaspor", "BS");
+
+    public static void main(String[] args) {
+
+        TeamArray teamArr = new TeamArray();
+        MatchArray matchArr = new MatchArray();
+        Test t = new Test();
```

Comment:
t mantıklı bir isimlendirme değil. mesela 'manager' kullanabilirsin. ya da başka bir şey. isimlendirmeler için chatgpt'den yardım isteyebilirsin :)